corgi-concourse-resource/tests/test_resource.py

```diff
@@ -167,34 +167,51 @@ def test_resource_files_are_written(self):
             job_json[key] = None
         assert job_json == expected_json
 
-        collection_id = read_file(os.path.join(dest_path, "collection_id"))
-        assert collection_id == read_file(os.path.join(DATA_DIR, "collection_id"))
+        repository = read_file(os.path.join(dest_path, "repo"))
+        assert repository == read_file(os.path.join(DATA_DIR, "repo"))
+
+        slugs = read_file(os.path.join(dest_path, "slugs"))
```

Comment:
:tada: So nice to get away from the `collection_id` file and have repo and slugs in their own file!